modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -614,16 +678,53 @@ return view.extend({
 			so.value(ipv4, ipaddrs[ipv4] ? '%s (%s)'.format(ipv4, ipaddrs[ipv4]) : ipv4);
 		});
 
-		so = ss.option(form.Value, 'leasetime', _('Lease time'));
+		so = ss.option(form.Value, 'leasetime', 
+			_('Lease time'),
+			_('Host-specific lease time, e.g. <code>5m</code>, <code>3h</code>, <code>7d</code>.'));
 		so.rmempty = true;
-
-		so = ss.option(form.Value, 'duid', _('DUID'));
+		so.value('5m', _('5m (5 minutes)'));
+		so.value('3h', _('3h (3 hours)'));
+		so.value('12h', _('12h (12 hours - default)'));
+		so.value('7d', _('7d (7 days)'));
+		so.value('infinite');
```

Comment:
A translation would make sense here, e.g. `_('infinite (lease does not expire)')`

Reply:
Good thinking...